docs/admin_api/rooms.md

```diff
@@ -171,3 +171,34 @@ Response:
 
 Once the `next_token` parameter is no longer present, we know we've reached the
 end of the list.
+
+# DRAFT: Room Details API
```

Comment:
Do you expect additional changes to this API?

Reply:
Yes, I made didn't add this to #5925 and now I'm not allowed to make any changes.
I think APIs need a bit of real world usage until they get stable.

Reply:
I don't think we have a precedent for draft admin APIs yet. Should we put them on `/_synapse/admin/unstable/...`?

Reply:
Probably not. I tried to use `v2_alpha` as it is done in the REST client API, but that approach got rejected.

Reply:
@anoadragon453 Any further thoughts here? One of the last comments hanging.

Reply:
There are for sure still some issues that will only manifest when we start using the API in clients. For example the users API still has issues with `displayname` vs `display_name` and `name` vs `user_id`.
For this endpoint I'm not 100% sure if it's a good idea to include the members array or if there should be a separate endpoint for it, e.g. `/_synapse/admin/v1/rooms/<room_id>/members`.

Reply:
This API also wouldn't handle a room with a large number of members well (since you cannot paginate as part of it). Having a separate API endpoint for it probably makes sense.

Reply:
OK, I will remove the members property in this PR (but keep the DRAFT comment).